src/Catalog/Persistence/AzureStorage.cs

```diff
@@ -112,21 +138,39 @@ private StorageListItem GetStorageListItem(IListBlobItem listBlobItem)
                     }
 
                     destinationStream.Seek(0, SeekOrigin.Begin);
-                    await blob.UploadFromStreamAsync(destinationStream, cancellationToken);
+                    await blob.UploadFromStreamAsync(destinationStream,
+                                                     accessCondition: null,
```

Comment:
Why do we need to pass in `null`? Can we just not pass anything in for these parameters?